src/openassetio-core/include/openassetio/Context.hpp

```diff
@@ -67,10 +89,29 @@ class OPENASSETIO_CORE_EXPORT Context final {
   /**
    * Describes what the @ref host is intending to do with the data.
    *
-   * For example, when passed to resolve, it specifies if the @ref
-   * host is about to read or write. When configuring a BrowserWidget,
-   * then it will hint as to whether the Host is wanting to choose a
-   * new file name to save, or open an existing one.
+   * For example, when passed to resolve, it specifies if the @ref host
+   * is about to read or write. When configuring a BrowserWidget, then
+   * it will hint as to whether the Host is wanting to choose a new file
+   * name to save, or open an existing one.
+   *
+   * When the access mode is one of the write patterns, the manager is
+   * expected to abide by the following procedure:
+   *
+   * 1. When the reference points to a non-existant entity, that entity
+   * should be created.
+   *
+   * 2. When the reference points to an existing entity:
+   *    * a. When the access is @ref Access.kCreateRelated a new
+   *      entity is created in relation to the target entity where
+   *      logical (eg: a child), or error is emitted.
+   *
+   *    * b. When the access is @ref Access.kWrite :
+   *        * i. When the trait set of the existing entity matches that
+   *          of the new entity, the entity should be updated (possibly
+   *          by versioning up). If this is not permitted, an error
+   *          should be emitted.
+   *        * ii. When the trait set of the existing entity does not
+   *          match, behave as per 2a.
    */
```

Comment:
[consider] Is it worth refing this to the publishing section in the interface docs, to avoid getting out of sync?

Reply:
Maybe a glossary link then? It feels like it is more important to describe this with all the other information about publishing etc? As these access modes don't just affect publishing etc (eg: getWithRelationship too)

Reply:
Maybe being so specific as to manager side implementation flow in the api docs is folly.

What do you think of this, just a section in the `register` documentation expanding on the previously existing section about lack of opinion?

https://github.com/OpenAssetIO/OpenAssetIO/pull/1036/commits/fffffcbc6fb529ceec9c17a5144fae6ef8df7377

Reply:
Perfec 👍 🙏